README.rst

```diff
@@ -59,3 +62,34 @@ This will create a temporary directory called tdir and cd into it then run both
 and put their results inside of their own directory named after themselves.
 
 From there you can explore them on your own
+
+Running via Docker
+^^^^^^^^^^^^^^^^^^
+
+In the below command we are instructing docker to mount the 
+`ngs_mapper/tests/fixtures/functional` directory from your local computer to 
+`/data` inside of the docker container when it runs.
```

Comment:
This makes it sound like they have to have this path 'ngs_mapper/tests/fixture/functional' on their computer already but this doesn't happen unless you install ngs_mapper which we are trying to avoid and use docker instead. So it needs to be clarified that this path that they should put here is path to their samples they want analyzed (ie. /NGSDATA)

Reply:
To do this...do they need to be 'in' their /NGSDATA directory when they run the command from docker? If so, please add that.